.github/workflows/run_doublet-detection.yml

```diff
@@ -43,11 +43,17 @@ jobs:
           conda activate openscpca-doublet-detection
           ./download-data.py --test-data --format SCE
 
-      - name: Run doublet-detection module
+      - name: Run doublet-detection module on benchmark test data
+        run: |
+          cd ${MODULE_PATH}
+          conda activate openscpca-doublet-detection
+          test=1 cores=1 ./run_doublet-detection-benchmark.sh
```

Comment:
I don't think you actually need to specify using 1 core? I use 4 in the ewing's action without an issue.

Reply:
> I use 4 in the ewing's action without an issue.

Ah cool, I'll remove all this